pkgs/tools/misc/turbo/default.nix

```diff
@@ -12,22 +13,23 @@
 , extra-cmake-modules
 , fontconfig
 , go
+, CoreServices
 }:
 let
-  version = "1.7.0";
+  version = "1.8.1";
   src = fetchFromGitHub {
     owner = "vercel";
     repo = "turbo";
     rev = "v${version}";
-    sha256 = "YTuEv2S3jNV2o7HJML+P6OMazgwgRhUPnd/zaTWfDWs=";
+    sha256 = "sha256-CkGi7qtFcioQAHeqKkHGrI/iMu8MnlK7IVmCB4AqIfo=";
```

Comment:
```suggestion
    hash = "sha256-CkGi7qtFcioQAHeqKkHGrI/iMu8MnlK7IVmCB4AqIfo=";
```

Reply:
Done in e1cce615a65c21adbc2bac3c4be79d2ae7b057b5